### PR TITLE
Makes Box's HOS office more secure

### PR DESCRIPTION
**Pull request by FloranOtten**
## About The Pull Request
Changes walls in Box HOS office to Reinforced walls, adds electricity to grilles.

## Why It's Good For The Game
Box's HOS locker was stupidly easy to steal. All you needed was a wrench and a welding tool, optionally a spacesuit. I've stolen the locker without any protection before by just rushing in and out of the nearby airlock. This PR aims to make it harder to steal the locker, as deconstructing a reinforced wall is more time consuming and more obvious. 

## Changelog
:cl:
tweak: Made HOS office more secure on Box
/:cl:

### CODE REVIEW

**Comment by Anonmare** [comment]
>inb4 "reeeeee sec buff"

**Comment by optimumtact** [comment]
you clearly don't understand the mindset of what box set out to achieve, but meta has rotted so many peoples brains that ultra secure maps with no weaknesses is apparently the new normal.

**Comment by Barhandar** [comment]
Where is the mapdiffbot

**Comment by OverlyGeneric** [comment]
> you clearly don't understand the mindset of what box set out to achieve, but meta has rotted so many peoples brains that ultra secure maps with no weaknesses is apparently the new normal.

It shouldn't be easy to get an antagonist objective in one of the most dangerous departments for antagonists to break into by welding a wall down and unwrenching a locker before escaping into space with locker in tow.

**Comment by optimumtact** [comment]
the question you have to ask yourself is: is my change actually improving the gameplay, or is it decreasing it

and so far, Im not convinced it's improving it.

**Comment by FloranOtten** [comment]
I don't consider Johnny Greyshirt having the seclathe and the HoS gun (should he have forgotten to take it out) to be good gameplay. It should be hard to nick these items, as right now it takes a welder and a wrench.

**Comment by Barhandar** [comment]
Move the locker as HoS then. Or get an engineer to reinforce your walls. Reinforcing them roundstart makes HoS room a second armory + lulls you into false sense of security.

**Comment by FloranOtten** [comment]
As for the notion that this is somehow transferring Meta's Hugbox, Box is the only map besides Pubby that has regular wall HoS office. And even Pubby's regularwalls can only be accessed by removing 4 or 5 (depending on your chosen path) walls in well travelled brig areas.

**Comment by anconfuzedrock** [comment]
box is the only map besides pubby... well, that's out of how many maps we currently use?
box
pubby
deltastation
metastation
donutstation

2/5 doesn't sound so bad. especially since deltastation is a complete rip off metastation and metastation setting the precedent is why there's so much excess security. "other maps do this" isn't an argument

this is a good pr though

**Comment by ArcaneMusic** [comment]
Hacking doors already allows for assistants,traitors and engineers to break into almost every secure area on most stations within 20 seconds or more. If you just want to tide the heads and steal their lockers for sport, you can already do this relatively easily from inside the station at low risk to yourself but at the expense of time. On box, being a flimsier station, it's possible to break into much, MUCH more secure areas than just the HOS office like this. You can use a welder and a wrench on 2 walls from the main hall and break into the bridge, and just one wall to break into the CMO and RD's office.

 So, why then does nobody do this, every round? The answer is that Box balances these weaker walls by making them higher visibility areas. The HOS office is visible to every officer in the locker room, the CMO's office is visible from the medical storage area and parts of cryo, RD's office is visable to anyone passing by toxins/xenobio and RnD. Now, I get that your argument is that because it's the brig, it should be a much more secure area considering it's right next to the armory, but most players don't break into the HOS office this way because it will ALMOST CERTAINLY alert EVERY OFFICER on station that you're breaking in.

So, tl;dr: I see where you're coming from but I feel like this goes against box's "flimsy deathtrap" philosophy.

**Comment by optimumtact** [comment]
@FloranOtten that is because every other map is heavily influenced by the massive amounts of hugbox that meta introduced to even get considered as a good map alongside the original box map.

**Comment by ExcessiveUseOfCobblestone** [comment]
> I don't consider Johnny Greyshirt having the seclathe, the HoS gun (should he have forgotten to take it out), the HoS hardsuit and the Security door remote to be good gameplay. 

The Sec Door remote is with the warden's locker isn't it?

The Sec lathe isn't in the HoS office?

**Comment by Barhandar** [comment]
>The Sec lathe isn't in the HoS office?

He means the spare board, which is in HoS's locker.

**Comment by optimumtact** [comment]
okay the real issue here is the HoS's locker is full of stupid shit

**Comment by ExcessiveUseOfCobblestone** [comment]
the techfab doesn't sync to the Ore Bank and the weapons still require a firepin so I don't see that as a big deal, not to mention you have to actually build it.

The only thing of note in HoS office is the gun, mindshields (during rounds where required), and maybe the space suit which you should already have if entering from space.

Let the warden handle it preemptively rather than giving them ample time to be reactionary when they've done nothing but sit in brig all round.

I plan on closing this unless someone else wants it.

**Comment by Okand37** [comment]
You could move the locker to the bottom left corner of the room, or simply mirror the room and it would make taking the locker a bit more of an involved process. Out of curiousity though, which windows are you intending to shock? The Head of Security's office already has shocked grilles (at least as of now, it might have been a more recent PR.)
